pkg/kubelet/kubelet_test.go

```diff
@@ -2408,14 +2408,17 @@ func TestSyncLabels(t *testing.T) {
 							t.Logf("No action yet")
 							return false, nil
 						}
-						action := actions[1]
-						if action.GetVerb() == "patch" {
-							patchAction := action.(core.PatchActionImpl)
-							var err error
-							savedNode, err = applyNodeStatusPatch(test.existingNode, patchAction.GetPatch())
-							if err != nil {
-								t.Logf("node patching failed, %v", err)
-								return false, nil
+						for _, action := range actions {
+							if action.GetVerb() == "patch" {
+								patchAction := action.(core.PatchActionImpl)
+								var err error
+								savedNode, err = applyNodeStatusPatch(test.existingNode, patchAction.GetPatch())
+								if err != nil {
+									t.Logf("node patching failed, %v", err)
+									return false, nil
+								}
+							} else {
+								t.Logf("got %v action when expecting a patch", action.GetVerb())
```

Comment:
this message seems misleading if we were previously skipping actions[0]

Reply:
I assumed actions[0] to be always `get` based on my local testing. Obviously it is the not the case. The main reason for putting a log is surface the action which can be ignored. If it is confusing, I can remove it.

Reply:
it's confusing because it looks like an error, when we'll spit out "got get action when expecting a patch" ~frequently in normal operations

Reply:
Fair enough, I've just removed the log statement.